Documentation/network/pod-mac-address.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _pod_mac_address:
+
+************************************
+Use a Specific MAC Address for a Pod
+************************************
+
+
+Some applications bind software licenses to network interface MAC addresses.
+Typically, the solution is very complex and requires things like Docker within
+the cluster.
+
```

Comment:
nit, I don't think that this sentence provides much clarity for the motivation of the feature. I think that the text would be just as clear without this.

```suggestion
```